src/FluentValidation/Validators/LengthValidator.cs

```diff
@@ -65,7 +65,16 @@ public LengthValidator(Func<object, int> min, Func<object, int> max)
 				context.MessageFormatter
 					.AppendArgument("MinLength", Min)
 					.AppendArgument("MaxLength", Max)
-					.AppendArgument("TotalLength", length);
+					.AppendArgument("TotalLength", length)
+					.AppendArgument("MinLengthCharacter", Min == 1

```

Comment:
This argument is not currently used in any message and was added to complete the set of available placeholders in the resource files.